src/CTA.Rules.Update/CTA.Rules.Update.csproj

```diff
@@ -13,6 +13,7 @@
     <ProjectReference Include="..\CTA.Rules.Models\CTA.Rules.Models.csproj" />
     <ProjectReference Include="..\CTA.Rules.ProjectFile\CTA.Rules.ProjectFile.csproj" />
     <ProjectReference Include="..\CTA.Rules.RuleFiles\CTA.Rules.RuleFiles.csproj" />
+    <ProjectReference Include="..\CTA.WebForms2Blazor\CTA.WebForms2Blazor.csproj" />
```

Comment:
Why do we need this reference? I thought we removed the dependency from the Update project.